### PR TITLE
Remove the SRAA institution switcher

### PR DESCRIPTION
**Pull request by pablothedude**
The SRAA (institution) switcher could be removed, because it is no longer useful after the FGA changes. This because earlier all overviews were from the viewpoint of an institution but now it's from a registration authority.

### CODE REVIEW

File: src/Surfnet/StepupRa/RaBundle/Form/Type/SearchRaCandidatesType.php [hunk 1]
```diff
@@ -55,6 +61,7 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
             'data_class' => 'Surfnet\StepupRa\RaBundle\Command\SearchRaCandidatesCommand',
+            'is_sraa' => false,
```
Comment (MKodde) [inline src/Surfnet/StepupRa/RaBundle/Form/Type/SearchRaCandidatesType.php]: What use does the is_sraa default value serve?

Comment (pablothedude) [reply src/Surfnet/StepupRa/RaBundle/Form/Type/SearchRaCandidatesType.php]: That should have been removed, was used as debug code. Removed

**Comment by pablothedude** [comment]
I did the translations but didn;t push the changes. The translations should be fixed now.

**Comment by MKodde** [comment]
Almost there, please fix the code sniffer issue that breaks the build.

**Pull request merged**